src/functions-and-arrays.js

```diff
@@ -1,41 +1,141 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
+function maxOfTwoNumbers(number1, number2) {
+  if (number1 > number2) {
+    return number1;
+  } else if (number2 > number1) {
+    return number2;
+  } else if (number1 === number2) {
+    return number1;
+  }
```

Comment:
Since you clearly understand your own code, some tips for you to refactor ;)

since we only check the length, below code would do the same:
```
  if (number1 >= number2) {
    return number1;
  } else if (number2 > number1) {
    return number2;
  }
```